### ISSUE TITLE
[Bug] Canvas size dosen't work properly with version 2.2.1

### ISSUE THREAD

**Issue by dreamsinmotion** [splinetool/react-spline#40]
I've updated to the new version 2.2.1 and in my existing project and in the example project from the repository the canvas is sized randomly with 0x0 pixel on load. I've tested this behaviour in Chrome, Safari and Firefox and got the same result.

https://user-images.githubusercontent.com/25156150/169344626-df4a14ce-86cc-43de-b815-5fb6ceffaf95.mp4

.

**Comment by marcofugaro** [comment]
@dreamsinmotion did you upgrade `@splinetool/runtime` to latest version as well?

**Comment by dreamsinmotion** [comment]
Yes, the runtime version is up to date. On the local dev server everything is running fine now, but when I build the project the strange behavior is back, ecpacialy in Chrome. The canvas is on page load 0x0 and only when I resize the window manually, the canvas is properly sized. I tried this just by simply using the export react function in the Spline app, downloaded the zip, install and build.

I'm getting this warnings from WebGL in the console:

<img width="811" alt="webglerrors" src="https://user-images.githubusercontent.com/25156150/169648162-e71a3fb6-c1a2-4c03-8850-ee77eccc0653.png">

**Comment by dreamsinmotion** [comment]
I tried a few workarounds and ended up with this. When I directly set the size on load without the timeout, I only get a black screen. With the timeout it works, quick and dirty.

```
function onLoad(spline)
{
      spline.setSize(0, 0);

      setTimeout(function () {
          spline.setSize(window.innerWidth, window.innerHeight);
      }, 100);
}
```

**Comment by marcofugaro** [comment]
@dreamsinmotion it should be fixed in version `0.9.58` of `@splinetool/runtime`, no need for the workaround anymore. Could you confirm?

**Comment by dreamsinmotion** [comment]
@marcofugaro I'm getting the same problem but less frequently and irregularly. 30% of the time its showing up on load but I still got the same WebGL errors before it renders something.

**Comment by marcofugaro** [comment]
Fixed with version `0.9.62` of `@splinetool/runtime`
